src/librustc/ty/mod.rs

```diff
@@ -2913,7 +2913,7 @@ impl<'a, 'gcx, 'tcx> TyCtxt<'a, 'gcx, 'tcx> {
 
     /// See `ParameterEnvironment` struct def'n for details.
     /// If you were using `free_id: NodeId`, you might try `self.region_maps.item_extent(free_id)`
-    /// for the `free_id_outlive` parameter. (But note that that is not always quite right.)
+    /// for the `free_id_outlive` parameter. (But note that is not always quite right.)
```

Comment:
Perhaps we should avoid the "note" entirely:

* Can we describe the circumstances under which it's not quite right?
* Why can't we rewrite the comment so it's always exactly right?